package.json

```diff
@@ -57,7 +57,6 @@
     "allure-js-commons": "^1.3.2",
     "arrify": "^2.0.1",
     "axios": "^0.19.1",
-    "acorn": "^7.1.0",
```

Comment:
nope, you need this
if you want to parse function arguments, acorn is required

Reply:
its duplicate. see above